.github/workflows/test.yml

```diff
@@ -1,5 +1,9 @@
 name: Test
-on: push
+on:
```

Comment:
This change isn't needed; the current CI works, it just hadn't been run since the most recent deno update.

Reply:
what causes it not to run?

Reply:
It only runs on push, so if code hasn't been pushed since a deno upgrade, there could be new uncaught TS errors.

Reply:
Turns out it was actually not running because it was from a fork.